src/IO/Exporter/Python/InterpolateToPoints.py

```diff
@@ -26,34 +27,7 @@ def parse_points(ctx, param, values):
 
 
 @click.command(name="interpolate-to-points")
-@click.argument(
-    "h5_files",
-    nargs=-1,
-    type=click.Path(exists=True, file_okay=True, dir_okay=False, readable=True),
-)
-@click.option(
-    "--subfile-name",
-    "-d",
-    help=(
-        "Name of subfile within h5 file containing volume data to interpolate."
-    ),
-)
-@click.option(
-    "--list-vars",
-    "-l",
-    is_flag=True,
-    help="Print available variables and exit.",
-)
-@click.option(
-    "--var",
-    "-y",
-    "vars",
-    multiple=True,
-    help=(
-        "Variables to interpolate. List any tensor components "
-        "in the volume data file, such as 'Shift_x'."
-    ),
-)
+@open_volfiles_command
 @click.option(
     "--target-coords-file",
     "-t",
```

Comment:
Can't reach it with Github comments, but replace the `parse_points` below in `target-coords` with the new one you wrote?